contrib/datawave-quickstart/bin/services/accumulo/bootstrap.sh

```diff
@@ -104,22 +104,20 @@ export PATH=${ACCUMULO_HOME}/bin:${ZOOKEEPER_HOME}/bin:$PATH
 
 DW_ZOOKEEPER_CMD_START="( cd ${ZOOKEEPER_HOME}/bin && ./zkServer.sh start )"
 DW_ZOOKEEPER_CMD_STOP="( cd ${ZOOKEEPER_HOME}/bin && ./zkServer.sh stop )"
-DW_ZOOKEEPER_CMD_FIND_ALL_PIDS="ps -ef | grep 'zookeeper.server.quorum.QuorumPeerMain' | grep -v grep | awk '{ print \$2 }'"
+DW_ZOOKEEPER_CMD_FIND_ALL_PIDS="ps -ef | grep 'zookeeper.server.quorum.QuorumPeerMain' | grep -v grep"
 
 DW_ACCUMULO_CMD_START="( cd ${ACCUMULO_HOME}/bin && ./accumulo-cluster start )"
 DW_ACCUMULO_CMD_STOP="( cd ${ACCUMULO_HOME}/bin && ./accumulo-cluster stop )"
 DW_ACCUMULO_CMD_FIND_ALL_PIDS="pgrep -d ' ' -f 'o.start.Main manager|o.start.Main tserver|o.start.Main monitor|o.start.Main gc|o.start.Main tracer'"
+DW_ACCUMULO_CMD_FIND_ALL_PIDS_COUNT="pgrep -f 'o.start.Main manager|o.start.Main tserver|o.start.Main monitor|o.start.Main gc' | wc -l"
 
 function accumuloIsRunning() {
-    DW_ACCUMULO_PID_LIST="$(eval "${DW_ACCUMULO_CMD_FIND_ALL_PIDS}")"
-
-    zookeeperIsRunning
-
-    [[ -z "${DW_ACCUMULO_PID_LIST}" && -z "${DW_ZOOKEEPER_PID_LIST}" ]] && return 1 || return 0
+    DW_ACCUMULO_PID_COUNT="$(eval "${DW_ACCUMULO_CMD_FIND_ALL_PIDS_COUNT}")"
+    [ "${DW_ACCUMULO_PID_COUNT}" -eq 4 ] && return 0 || return 1
```

Comment:
I feel like it's probably not a good idea to have running/stopped status depend on the presence of a specific number of pids. The original goal was that the presence of any pids for a service should signal 'running' status, and for the pids themselves to be collected uniformly into a ```*_PID_LIST```  variable, so that you can easily ```kill -9``` them and/or report them to stdout on demand, if needed

That's not to say what's currently here is bulletproof by any means.

Could you could you provide more detail about the specific issue(s) you encountered when trying to start/stop things? That would help me better understand how we might fix things. 

One thing I've considered is breaking out Zookeeper as its own top-level service, rather than have it be embedded with Accumulo. That could go a long way toward simplifying things and probably squash some existing bugs here.